mrjob/cloud.py

```diff
@@ -61,6 +62,7 @@ class HadoopInTheCloudJobRunner(MRJobBinRunner):
         'check_cluster_every',
         'cloud_fs_sync_secs',
         'cloud_tmp_dir',
+        'cloud_upload_part_size',
```

Comment:
cloud_upload_part_size_mb  ?

I didn't see any docs anywhere stating the units of this param